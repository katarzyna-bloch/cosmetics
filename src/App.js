import React from 'react'
import styled from 'styled-components'

const Header = styled.header`
  top: 0;
  left: 0;
  position: sticky;
  background-color: palevioletred;
`

const Main = styled.main`
  flex: 1;
  align-self: center;
  max-width: 1024px;
  padding: 10px;
`

const Footer = styled.footer`
  background-color: palevioletred;
`

function App() {
  return (
    <>
      <Header>...</Header>
      <Main>content</Main>
      <Footer>...</Footer>
    </>
  );
}

export default App;
