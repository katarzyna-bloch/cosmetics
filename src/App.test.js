import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

it('renders correctly', () => {
  let wrapped = shallow(<App />);
  expect(toJson(wrapped)).toMatchSnapshot();
});
