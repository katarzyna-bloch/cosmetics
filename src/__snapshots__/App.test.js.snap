// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <styled.header>
    ...
  </styled.header>
  <styled.main>
    content
  </styled.main>
  <styled.footer>
    ...
  </styled.footer>
</Fragment>
`;
